src/utils/helpers/createEvent/index.js

```diff
@@ -0,0 +1,25 @@
+/**
+ * Override a SyntheticEvent with a custom event.target
+ * This is intended to be used by Carbon components that provide a formatted value or those that need
+ * to provide a name or value that does no appear in the DOM e.g. a component that has two controls associated with one
+ * input.
+ */
+export default (event, overrides) => {
```

Comment:
Could you change that helper to a named function? When it's anonymous it's difficult to find in an IDE, you also cannot auto import it in VSCode.